iroha/src/sumeragi.rs

```diff
@@ -31,59 +35,126 @@ trait Consensus {
 #[derive(Io, Decode, Encode, Debug, Clone)]
 pub enum Message {
     /// Is sent by leader to all validating peers, when a new block is created.
-    Created(SignedBlock),
+    CreatedBlock(SignedBlock),
     /// Is sent by validating peers to proxy tail and observing peers when they have signed this block.
-    Signed(SignedBlock),
+    SignedBlock(SignedBlock),
     /// Is sent by proxy tail to validating peers and to leader, when the block is committed.
-    Committed(SignedBlock),
+    CommittedBlock(SignedBlock),
     /// Is sent when the node votes to change view due to commit timeout.
     CommitTimeout(CommitTimeout),
+    /// Receipt of receiving tx from peer. Sent by a leader.
+    ReceivedTx(TxReceipt),
+    /// Tx forwarded from client by a peer to a leader.
+    ForwardedTx(ForwardedTx),
+    /// Message to other peers that this peer did not receive receipt from leader for a forwarded tx.
+    NoTxReceipt(NoTxReceipt),
 }
 
 impl Message {
     async fn send_to(self, peer: &PeerId) -> Result<(), String> {
         let _response = Network::send_request_to(
             &peer.address,
-            Request::new(uri::BLOCKS_URI.to_string(), self.into()),
+            Request::new(uri::CONSENSUS_URI.to_string(), self.into()),
         )
         .await?;
         Ok(())
     }
 }
 
+/// Message structure describing a failed attempt to forward tx to a leader.
```

Comment:
We have too much messages in the top module, move everything message related into `sumeragi::message` submodule.

Reply:
I guess we should do it in a separate refactoring Pull Request

Reply:
ok